# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/DeleteRangeTransaction.cpp
# Commit: f2f431ee78a3
# Full Hash: f2f431ee78a35a577f8d89a469ab17c20f71e3f0
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-09-21 03:56:08
# Regressor Bug: 1789967
# File Overlap Count: 1
# Description:
#   Bug 1789967 - part 2: Make `TextEditor` and `HTMLEditor` implement `EditorBase::CollapseSelectionToEndOfLastLeafNode` by themselves r=m_kato
#   
#   It does different thing for `TextEditor` and `HTMLEditor`, and used almost
#   internally.  Therefore, it should be implemented in the sub classes and
#   we should name them better.
# ==============================================================================

diff -r edac234f1eba -r f2f431ee78a3 editor/libeditor/DeleteRangeTransaction.cpp
--- a/editor/libeditor/DeleteRangeTransaction.cpp	Wed Sep 21 00:20:26 2022 +0000
+++ b/editor/libeditor/DeleteRangeTransaction.cpp	Wed Sep 21 00:38:23 2022 +0000
@@ -119,9 +119,8 @@
 
   OwningNonNull<EditorBase> editorBase = *mEditorBase;
   rv = editorBase->CollapseSelectionTo(EditorRawDOMPoint(startRef));
-  NS_WARNING_ASSERTION(
-      NS_SUCCEEDED(rv),
-      "EditorBase::CollapseSelectionToEndOfLastLeafNode() failed");
+  NS_WARNING_ASSERTION(NS_SUCCEEDED(rv),
+                       "EditorBase::CollapseSelectionTo() failed");
   return rv;
 }
 